tensorflow_datasets/audio/freesound_audio_tagging_2019.py

```diff
@@ -0,0 +1,152 @@
+from pathlib import Path
+
+import kaggle
+import pandas as pd
```

Comment:
Please have a look at lazy_imports for this: https://www.tensorflow.org/datasets/add_dataset#extra_dependencies